vtex-apps/trustpilot
# Replace micro count with generic widget

**What problem is this solving?**

Add support for different trustpilot widgets.

**How should this be manually tested?**

Go to https://kiwi--miriadeit.myvtex.com/ and check the footer

Compare with https://miriadeit.myvtex.com/

**Screenshots or example usage:**

![image](https://user-images.githubusercontent.com/12702016/91334062-d0165a00-e7a4-11ea-93dc-a9b807aae309.png)
